Aptoide/appcoins-wallet-android
# [APPC-1248] Implement promotions screen

**What does this PR do?**

 Adds the promotion screen and the overlay view for the promotions bottom navigation.
A refactor on the gamification progress bar was also done, extracting it to a custom view
Note: There’s still no Api calls on this PR
**Database changed?**

No

**Where should the reviewer start?**

PromotionsFragment.kt
OverlayFragment.kt
GamificationProgressBarView.kt

**How should this be manually tested?**

  Check if the updates notification work ok (according to the mocked values)
  Check if the gamification progress bar increases the first time both in the promotions view as well as the gamification view
  Check if the overlay fragment works well

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/APPC-1248

**Code Review Checklist**
